src/configure/configure.ts

```diff
@@ -128,6 +128,8 @@ class Orchestrator {
     private async getAzureResource(targetType: TargetResourceType) {
         var azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(targetType);
         this.inputs.targetResource.resource = await azureResourceSelector.getAzureResource(this.inputs);
+        telemetryHelper.setTelemetry(TelemetryKeys.resourceType, this.inputs.targetResource.resource.type);
```

Comment:
weren't these being already logged in some or other form?

Reply:
These are being logged in right click scenario, but were not there in the other flow. So added.